mobile/function/map_page_function.js

```diff
@@ -32,13 +32,13 @@ const GetMapPage = (routeId, hasLocation) => {
 
         const json = await response.json();
         setData(json);
-        // console.log(data.route);
+        console.log("we komen bij de fetchdata");
```

Comment:
Deze console.logs moeten er of uit of verandertworden naar een zin, die logisch is.